db/schema.rb

```diff
@@ -16,6 +16,16 @@
   enable_extension "pgcrypto"
   enable_extension "plpgsql"
 
+  # These extensions are not used by our app, but they are always enabled for
```

Comment:
Will Rails remove these comments the next time `schema.rb` is regenerated by `db:migrate`? It might be better to put these notes in the commit message instead.

Reply:
Very good point, I didn't realise that the file is completely overwritten by Rails. Yeah, I'll move it to the commit message.

Reply:
(Clearly I didn't test it well enough, either...)

Reply:
Keeping comments such as these in the commits rather than files also serves to keep the general noise and distractions to a minimum. We trust that we'll be able to interrogate lines to find out more if and when we need them through their commits.

Reply:
Moved it to the commit message.